_posts/2024-02-18-how-its-made.md

```diff
@@ -0,0 +1,180 @@
+Bug fixes are still applied to the branched code, but new features are not.
+This helps to prevent last-minute problems in the release versions.
+
+After branching, the testing releases continue with one or more Release Candidates (RCs).
+RCs are similar to the beta version in that issues are still expected, but as no new features are applied anymore, the issue count should go down, not up.
+The actual release is basically done the same way as a beta: prepare a changelog, all the social news, make a tag, wait for the CI, and finally tell the world.
+
+When no more major issues are found in the latest RC, the time for the first proper release has come.
+There's really not much difference in the release process from the RCs, except that this version is not marked as a testing release and thus will be for example picked up by the automatic update on Steam or GOG.
+We do spend more time on the changelog, news post and social media posts for the release, as a proper non-testing release has a much larger audience than the preceding testing releases.
+
+Inevitably, a larger audience is also better in finding bugs, which is why the maintenance releases exists.
+Maintenance releases are the X.1, X.2 and so on versions.
+They do not include new features over the major X.0 release, but fix whatever issues are found.
+
+Thanks to a lot of behind the scenes magic done by some wizards on the CI workflows, the most time-consuming part of any release is actually creating and publishing all the public-facing text and information.
+Compiling the binary and uploading the result to the various distribution platforms requires surprisingly few mouse clicks to trigger.
+This wasn't always the case though.
+When OpenTTD was still young and sites like GitHub were just not a thing, the various binaries for e.g. Windows, Linux or macOS were hand-compiled by different people, manually collected and uploaded.
+Often enough that meant that it literally took several days until a release could be downloaded for all platforms.
+Compared to that, the current automatic CI workflows really do feel like awesome magic.
+
+
+## Getting personal
+
+If you've been diligently reading till now, you might have noticed that haven't used the word "developer" so far.
+Instead, I've mostly talked about a generic someone.
+So what's up with that?
+
+![Contributors to OpenTTD]({% link /static/img/post_2024-02-18-how-its-made/contributors.png %}){:width="300px" style="float:right; padding:10px"}
+
+While many people think that the individuals with merge permissions are "the" developers of OpenTTD, this isn't really true at all.
+GitHub provides various statistics for each project, and lists 169 contributors to OpenTTD at the time of writing this.
+And this number is still much too low, as it is for example missing many contributors from before we were on GitHub or our language translators.
```

Comment:
```suggestion
And this number is still much too low, as it is for example missing many contributors from before we were on GitHub, and doesn't count our language translators since their changes are committed by a bot.
```